### ISSUE TITLE
Go all in on plain text - swap out `sanitize-html` with a plain text sanitizer

### ISSUE THREAD

**Issue by mwickett** [my-covid-story/www#137]
I reached for `sanitize-html` without fully thinking it through - we're actually not using html at all with user generated content - it's all plain text. So we should sanitize plain text only.

Potential options:
* [`sanitize`](https://www.npmjs.com/package/sanitize)
* [`html-to-text`](https://github.com/html-to-text/node-html-to-text)

**Comment by LuckeeDev** [comment]
Why do we need to sanitize the user generated content? I think that as long as we don't pass anything inside `dangerouslySetInnerHTML` we're safe, but I might be wrong.

**Comment by mwickett** [comment]
Yeah, it's a fair question. Like you say, if we're not actually rendering the markup, it's not a security concern. But if someone _does_ add tags etc, it'll look bad.
